rules-reviewed/rhr/springboot/springboot-associated-artifacts.rhamt.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0"?>
+<ruleset id="springboot-associated" xmlns="http://windup.jboss.org/schema/jboss-ruleset"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            Verify the version of the Spring Boot framework is compatible with those supported by Red Hat Runtimes
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,3.0.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-java,3.0.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-xml,3.0.0.Final" />
+        </dependencies>
+        <sourceTechnology id="springboot" />
+        <targetTechnology id="rhr"/>
+    </metadata>
+    <rules>
+        <rule id="springboot-associated-00001">
+            <when>
+                <dependency groupId="org.amqphub.spring" artifactId="amqp-10-jms-spring-boot-starter" toVersion="2.0.9" />
+            </when>
+            <perform>
+                <hint title="Unsupported version of AMQP SpringBoot starter" effort="1" category-id="potential">
+                    <message>Unsupported version of AMQP SpringBoot starter</message>
```

Comment:
Change the text for this and the following messages to be like:
```suggestion
                    <message>Check the links below for the Red Hat Runtimes supported version of AMQP SpringBoot starter</message>
```

Reply:
I have altered the messages as suggested